addons/zeus/functions/fnc_ui_ToggleNvg.sqf

```diff
@@ -0,0 +1,122 @@
+/*
+ * Author: alganthe
+ * Initalises the `Toggle NVGs` zeus module display
+ *
+ * Arguments:
+ * 0: Nvg toggle controls group <CONTROL>
+ *
+ * Return Value:
+ * Nothing
+ *
+ * Example:
+ * onSetFocus = "_this call ace_zeus_fnc_ui_toggleNvg"
+ *
+ * Public: No
+*/
+
+#include "script_component.hpp"
+
+disableSerialization;
+
+params ["_control"];
+
+//Generic Init:
+private _display = ctrlparent _control;
+private _ctrlButtonOK = _display displayctrl 1; //IDC_OK
+private _logic = GETMVAR(BIS_fnc_initCuratorAttributes_target,objnull);
+TRACE_1("logicObject",_logic);
+
+_control ctrlRemoveAllEventHandlers "setFocus";
+
+// Handles errors
+private _unit = effectiveCommander (attachedTo _logic);
+
+scopeName "Main";
+private _fnc_errorAndClose = {
```

Comment:
this and the switch a few lines below is used in at least two places. Maybe make it a separate function?